sites/fast-creator/app/creator.tsx

```diff
@@ -37,17 +35,23 @@ import {
     ThemeSelector,
 } from "@microsoft/site-utilities";
 import { fastDesignSystemDefaults } from "@microsoft/fast-components/src/fast-design-system";
-import { StandardLuminance } from "@microsoft/fast-components";
+import { neutralLayerL1, StandardLuminance } from "@microsoft/fast-components";
 import * as monaco from "monaco-editor/esm/vs/editor/editor.api";
 import { monacoAdapterId } from "@microsoft/fast-tooling/dist/esm/message-system-service/monaco-adapter.service";
-import { CreatorState, ProjectFile } from "./creator.props";
+import { CreatorState, FormId, ProjectFile } from "./creator.props";
 import { divTag, linkedDataExamples } from "./configs";
 import { ProjectFileTransfer } from "./components";
 import { previewReady } from "./preview";
 import { Footer } from "./site-footer";
-import { renderDeviceSelect, renderDevToolToggle } from "./web-components";
+import {
+    renderDeviceSelect,
+    renderDevToolToggle,
+    renderFormTabs,
+} from "./web-components";
 import { Device } from "./web-components/devices";
 
+StandardLuminance;
+
```

Comment:
This is not a web component, it's an enum so we don't need to worry about tree shaking.

```suggestion
```

Reply:
Done